_posts/0000-01-02-reggiekcollins.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Use the left arrow to go back!
```

Comment:
Add a back arrow at the end of the sentence use the left arrow to go back  " < "